src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js

```diff
@@ -21,9 +19,8 @@ const ReportsDailyVisits = ({ domainName, dateFrom, dependencies: { datahubClien
       x: {
         type: 'timeseries',
         tick: {
-          format: (x) => {
-            return intl.formatDate(x, { timeZone: 'UTC' });
-          },
+          culling: false,
+          format: '%b %d',
```

Comment:
What is `%b`, does it not depend on current language?

Reply:
%b shows the monthly name with 3 chars

Reply:
But the first chars change between language, right?

Reply:
now yes! 
![image](https://user-images.githubusercontent.com/8861133/64428447-1e10ea00-d08a-11e9-81d5-1c041a2fb973.png)